src/components/victory-axis.jsx

```diff
@@ -99,6 +99,7 @@ class VictoryAxis extends React.Component {
     if (this.props.labelPadding) {
       return this.props.labelPadding;
     }
+    // TODO: magic numbers
```

Comment:
Ha. Neeeeeet. Question: d'you think it would serve us to pull magic numbers into `const`s at the top of the file? Make 'em easy to find and change, in case we ever want to set 'em programmatically or fiddle with em?